src/engraving/libmscore/stringdata.cpp

```diff
@@ -427,11 +427,11 @@ int StringData::fret(int pitch, int string, int pitchOffset) const
 {
     int strings = stringTable.size();
     if (strings < 1) {                          // no strings at all!
-        return FRET_NONE;
+        return INVALID_FRET_INDEX;
```

Comment:
You forgot to addjust the comment above

Reply:
same for `StringData::fret()` further up

Reply:
Thanks, I'll fix it!